app/controllers/admin/backups_controller.rb

```diff
@@ -44,9 +45,22 @@ def cancel
     render json: success_json
   end
 
-  # download
-  def show
+  def email
     if backup = Backup[params.fetch(:id)]
+      token = EmailBackupToken.set
+      hostname = Discourse.current_hostname
+      base_url = Discourse.base_url
+      download_url = "#{Discourse.base_url}/admin/backups/#{backup.filename}?token=#{token}"
+      Jobs::DownloadBackupEmail.new.execute(to_address: current_user.email, backup_file_path: download_url)
+      render nothing: true
+    else
+      render nothing: true, status: 404
+    end
+  end
+
+  def show
```

Comment:
We should probably generate a different view if the token is invalid or not present. Currently we're just returning a 404 which may be confusing to users.